ParquetClassLibrary/Sandbox/MapRegionEditor.cs

```diff
@@ -40,10 +40,11 @@ internal ParquetMask ParquetPaintPattern
             }
         }
 
-        private Floors _floorToPaint;
-        private Blocks _blockToPaint;
-        private Furnishings _furnishingToPaint;
-        private Collectables _collectableToPaint;
+        // TODO Add bounds-checking using the ranges defined in Assembly.  Put it in TrySet*
```

Comment:
Acknowledging the comment here; Panda had a similar thought that this'd be necessary with the new signatures letting you give a Furnishing to SetFloorToPaint, etc